jpsreport/general/ArgumentParser.cpp

```diff
@@ -102,11 +102,11 @@ void Logs()
     // first logs will go to stdout
     Log->Write("----\nJuPedSim - JPSreport\n");
     Log->Write("Current date   : %s", currentTime.c_str());
-    // Log->Write("Version        : %s", JPSREPORT_VERSION);
+    Log->Write("Version        : %s", JPSREPORT_VERSION);
     Log->Write("Compiler       : %s (%s)", true_cxx.c_str(), true_cxx_ver.c_str());
-    // Log->Write("Commit hash    : %s", GIT_COMMIT_HASH);
-    // Log->Write("Commit date    : %s", GIT_COMMIT_DATE);
-    // Log->Write("Branch         : %s", GIT_BRANCH);
+    Log->Write("Commit hash    : %s", GIT_COMMIT_HASH);
```

Comment:
Is there a _tag_ mistake?
`Commit hash    : v0.8.4-881-gb6522ee5`

Reply:
It is the same pattern as used in jps-report
Read it as:
{last tag}-{commits since last tag}-{short commit id}

Reply:
ok. Thanks.